load_test/xmpp_amqp.erl

```diff
@@ -242,11 +242,19 @@ xmpp_do(#{neighborIDs := NeighborIDs,
             send_presence_available(Client);
         {scheduled, send_message} ->
             schedule(send_message, SendMessageAfter),
-            send_messages_to_neighbors(Client, NeighborIDs)
+            send_messages_to_neighbors(Client, NeighborIDs);
+        {system, {From, Mref}, get_state} -> % allows to do sys:get_state(Pid)
+            From ! {Mref, S}
     end,
     xmpp_do(S).
 
 -spec process_stanza(exml:element()) -> ok.
+process_stanza(#xmlel{name = <<"message">>, attrs = A}) ->
```

Comment:
What is stanza is not a `message`?

Reply:
If stanza is not the message, it will match into second clause, which literally does nothing.

Apart from that there is `escalus_connection:set_filter_predicate(Client, fun escalus_pred:is_chat_message/1)` in `xmpp_start/1`.